src/herder/HerderImpl.cpp

```diff
@@ -949,17 +1006,25 @@ HerderImpl::triggerNextLedger(uint32_t ledgerSeqToTrigger,
     {
         nextCloseTime = lcl.header.scpValue.closeTime + 1;
     }
+    if (nextCloseTime < minCloseTime)
+    {
+        nextCloseTime = minCloseTime;
+    }
 
-    // Ensure we're about to nominate a value with valid close time
-    auto isCtValid =
-        ctValidityOffset(nextCloseTime) == std::chrono::milliseconds::zero();
-
-    if (!isCtValid)
+    auto const enhancedManualClose = mApp.getConfig().ENHANCED_MANUAL_CLOSE;
```

Comment:
I do not like this:
1. either we have full fidelity with `triggerNextLedger` (so no need to diverge from the existing code), or
2. we hook things up at the `ledgerClose` layer and aim to get partial fidelity
  * we're bypassing SCP but when there is a single value, this may not be that much of a big deal and the goal is to force the close time to take on a specific value anyways

"1" seems to be the best, but for that we need https://github.com/stellar/stellar-core/issues/1760
* any other way will require special handling (what you're doing here, and more) as the close time specified may not be compatible with the current date (when using non virtual time), and we need values to be valid during the SCP round (the part that you missed) - there might be even more special cases, and we don't want to go chase this stuff.
* a potential downside with this approach is that the "close time" specified may or may not be the actual time in the resulting ledger (as we'd be using all rules)

"2" will require extracting parts of `triggerNextLedger` so that we can share the code that calculates the candidate value, but instead of using SCP we would just invoke the close ledger code directly.

Both of those approaches require some special code that ensures that lcl is the "right one" when the ledger seqnum jumps ahead, otherwise SCP won't close that ledger and ledger close will crash (because it increments the ledger number by 1).

Reply:
I thought that enforcing `RUN_STANDALONE` would allow us to set the sequence number forward arbitrarily since there would be no nodes proposing different ones.  I didn't realize that it was automatic that each close just incremented the number.

I _think_ we want advancing the sequence number to be one of the things that we're letting people test, right?  Are you saying we'll need to change the sequence number in the header in `triggerNextLedger()` if it jumps ahead because of a manual request, to satisfy SCP?

Are you also saying that we should force the use of virtual time when MANUAL_CLOSE is specified (issue #1760) as part of the work for this issue?

Reply:
I'm now changing the sequence number in a separate modification before `triggerNextLedger()`, and I am indeed using `VIRTUAL_TIME` when `MANUAL_CLOSE` is specified, as suggested by issue #1760 .